app/src/main/java/com/keylesspalace/tusky/ComposeActivity.java

```diff
@@ -578,6 +593,45 @@ public void afterTextChanged(Editable s) {
         }
 
         textEditor.requestFocus();
+
+        if (inReplyToId == null) {
+            composeAvatar.setOnClickListener(v -> showAccountSwitchDialog(true));
+
+            if (!intent.hasExtra(ACCOUNT_PRESET) && !intent.hasExtra(SAVED_TOOT_UID_EXTRA) && (getCallingActivity() == null || !getCallingActivity().getClassName().equals(MainActivity.class.getName()))) {
```

Comment:
Super long line :sweat_smile:
If you are to to check calling activity, maybe you should check that the caller is *not* `ComposeActivity`?